src/functions/findFirst.js

```diff
@@ -149,20 +150,19 @@ export const findFirst = wrap({
             hi.assert(result.high === 4);
         },
         "emptySearchString": (hi) => {
+            hi.assertUndefined(hi("test").findFirst(""));
+        },
+        "unboundedSearchInput": (hi) => {
             hi.assertFailWith(NotBoundedError, () => {
-                return hi("test").findFirst("");
+                return hi("test").findFirst(hi.repeatElement({
+                    key: "hello",
+                    value: "world"
+                }));
```

Comment:
Can we just like, `hi.repeatElement("x")` instead? This seems unnecessarily hard on the reader trying to understand what the test does